docs/en/sql-reference/statements/kill.md

```diff
@@ -21,6 +21,35 @@ The queries to terminate are selected from the system.processes table using the
 
 Examples:
 
+First, you'll need to get the list of incomplete queries. This SQL queries provides them according to those running the longest:
+
+List from a single ClickHouse node:
+``` sql
+SELECT
+  initial_query_id,
+  query_id,
+  formatReadableTimeDelta(elapsed) AS time_delta,
+  query,
+  *
+  FROM system.processes
+  WHERE query ILIKE 'SELECT%'
+  ORDER BY time_delta DESC;
+```
+
+List from a ClickHouse cluster:
+``` sql
+SELECT
+  initial_query_id,
+  query_id,
+  formatReadableTimeDelta(elapsed) AS time_delta,
+  query,
+  *
+  FROM clusterAllReplicas(default, system.processes)
```

Comment:
By default the `default` cluster consists of the local node only. If your `default` database has `Replicated` engine - then yes, the cluster will contain all the nodes on which this database is present.

Reply:
Is what you mean we should use `cluster` instead of `clusterAllReplicas` when using a non-`Replicated` engine?

Should we add an additional list query for clusters using `Replicated` engine and clusters using non-`Replicated` engines?